Scroller/EditableWrapper.js

```diff
@@ -89,6 +97,13 @@ const EditableWrapper = (props) => {
 
 		// Last InputType which moves Items
 		lastInputType: null,
+
+		// Timer for holding key input
+		timer: null,
```

Comment:
```suggestion
		keyHoldTimer: null,
```
I'm not sure... 
Please more specific name.

Reply:
@jeonghee27 How about `keyHoldTimerId`?
and `needToPropagateEvent` instead of `stopPropagationFlag` ?

Reply:
OK . `keyHoldTimerId`
`needToPropagateEvent` better I think

Reply:
It's fixed.